### PR TITLE
Swagger schema validation failure for plugin apis - Closes #5730

### LINKED ISSUE #5730: Swagger schema validation failure for plugin apis
### Expected behavior
All the swagger-schema must be properly validated. 

### Actual behavior

Some of the swagger parameters have wrong schema, causing issues while using different tools based on swagger. 

### Which version(s) does this affect? (Environment, OS, etc...)
5.0.0

### PR DESCRIPTION
**Pull request by nazarhussain**
### What was the problem?

This PR resolves #5730

### How was it solved?

Fixed the linting issues.

### How was it tested?

Validate swagger files using external tools.

### CODE REVIEW

**Review by pablitovicente — APPROVED** [review]

**Comment by pablitovicente** [comment]
LG; deep scan alert needs to be addressed though.

**Comment by nazarhussain** [comment]
@pablitovicente Yes I checked. The file deep scan is complaining about is not included in this PR. May be it was committed and merged earlier.

File: framework-plugins/lisk-framework-http-api-plugin/swagger.yml [hunk 1]
```diff
@@ -683,9 +747,12 @@ definitions:
...
         type: string
         description: Hex encoded value of the unique identifier of the transaction.
-      type:
+      moduleID: 
+        type: number
+        description: Module Id for transaction asset
+      assetID: 
         type: number
-        description: Describes the Transaction type.
+        description: Asset Id for for transaction asset
```
Comment (ManuGowda) [inline framework-plugins/lisk-framework-http-api-plugin/swagger.yml:755]:
```suggestion
        description: Asset Id for transaction asset
```

Comment (ManuGowda) [reply framework-plugins/lisk-framework-http-api-plugin/swagger.yml:755]: same for below as well

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**